ops/README.md

```diff
@@ -3,4 +3,4 @@
 
 This directory contains files related to deploying or running the application(s).
 
-Documentation is in [docs/ops](../docs/ops).
\ No newline at end of file
+Documentation is in [docs/ops](../docs/operations/).
```

Comment:
Thank you for changing this, I noticed it was in error the other day but didn't take the time to fix it.